mysql-test/suite/binlog_encryption/t/binlog_encryption_on_while_binlog_off.test

```diff
@@ -0,0 +1,28 @@
+# PS-287: It is possible to enable encrypt_binlog with no binary logging
+#
+# It is possible to enable encrypt_binlog without binary logging. In this setup
+# server will only encrypt relay logs - if used. We print an information in server's log
+# file to notify user that although encrypt_binlog in on, there is no binlog file to
+# encrypt and only relay logs will get encrypted.
+
+--let restart_hide_args=1
```

Comment:
`restart_hide_args` doesn't have to be declared as an environment variable, so here it should be `$restart_hide_args`.
Minor comment. Can be ignored.